src/tm_mad/ceph/postmigrate

```diff
@@ -36,6 +36,21 @@ TEMPLATE_64=$6
 
 #--------------------------------------------------------------------------------
 
+# Get custom datastore values
+DRIVER_PATH=$(dirname "$0")
+XPATH="${DRIVER_PATH}/../../datastore/xpath.rb --stdin"
+
+unset XPATH_ELEMENTS i j
+
+while IFS= read -r -d '' element; do
+  XPATH_ELEMENTS[i++]="$element"
+done < <(onedatastore show -x "$DSID" | $XPATH \
+  /DATASTORE/TEMPLATE/SHARED)
+
+SHARED="${XPATH_ELEMENTS[j++]:-$SHARED}"
```

Comment:
Excuse me if I am wrong but the SHARED attribute in the datastore template is set via the TM_MAD_CONF definition for the datastore in /etc/oned.conf so it is hard to have missing variable in the template. In addition the bash substitution for `:-$SHARED` expect to have `SHARED` in the environment which I think is not defined in ONE.